test/testcafe/spec/EnrollAuthenticatorSecurityQuestion_spec.js

```diff
@@ -0,0 +1,86 @@
+import { RequestMock, RequestLogger } from 'testcafe';
+
+import SuccessPageObject from '../framework/page-objects/SuccessPageObject';
+import EnrollSecurityQuestionPageObject from '../framework/page-objects/EnrollSecurityQuestionPageObject';
+
+import xhrAuthenticatorEnrollSecurityQuestion from '../../../playground/mocks/data/idp/idx/authenticator-enroll-security-question';
+import success from '../../../playground/mocks/data/idp/idx/success';
+
+const authenticatorRequiredSecurityQuestionMock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrAuthenticatorEnrollSecurityQuestion)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge/answer')
+  .respond(success);
+
+const answerRequestLogger = RequestLogger(
+  /idx\/challenge\/answer/,
+  {
+    logRequestBody: true,
+    stringifyRequestBody: true,
+  }
+);
+
+fixture(`Enroll Security Question Form`);
+
+async function setup(t) {
+  const challengeFactorPage = new EnrollSecurityQuestionPageObject(t);
+  await challengeFactorPage.navigateToPage();
+  return challengeFactorPage;
+}
+
+test.requestHooks(answerRequestLogger, authenticatorRequiredSecurityQuestionMock)(`enroll security question`, async t => {
+  const challengeFactorPageObject = await setup(t);
+
+  const radioOptionLabel = await challengeFactorPageObject.clickChooseSecurityQuestion();
+  await t.expect(radioOptionLabel).eql('Choose a security question');
+  await challengeFactorPageObject.selectSecurityQuestion(1);
```

Comment:
Can we add a check to make sure the non relevant fields are hidden in both the tests. 

eg) In this test the custom question field should be hidden,

Reply:
Good point! Thanks!